flocker/node/agents/blockdevice.py

```diff
@@ -956,7 +956,7 @@ class MandatoryProfiles(Values):
     GOLD = ValueConstant(u'gold')
     SILVER = ValueConstant(u'silver')
     BRONZE = ValueConstant(u'bronze')
-    DEFAULT = SILVER
+    DEFAULT = ValueConstant(BRONZE.value)
```

Comment:
mentioned this in original profiled implementation, but gold, sillver, bronze suggests some type of limit. Where as ```default``` typically zeros out values and allows "unlimited" style for default profile. Suggesting we might want to keep default actually default, instead of assigning it to gold, silver or bronze.